.github/workflows/config.yml

```diff
@@ -0,0 +1,32 @@
+name: Test Data Logger and publish
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    env:
+      working-directory: ./data_logger
+    steps:
+    - name: Set up Elixir
+      uses: erlef/setup-beam@v1
```

Comment:
I am really starting to love using this because the setup is waaaay faster because it has cached runners with this, while our container was built every run